jovian/callbacks/fastai_callback.py

```diff
@@ -0,0 +1,54 @@
+from fastai.basic_train import Learner
+from fastai.callback import Callback
+
+from jovian import log_hyperparams, log_metrics
+from jovian.utils.logger import log
+
+
+class JovianFastaiCallback(Callback):
+    """FastAI callback to log hyperparameters and metrics during model training.
+
+    Arguments:
+        learn (Learner): A learner object with which you're fitting your model
+
+        arch_name (string): A name for the architecture that you're using 
+    """
+
+    def __init__(self, learn: Learner, arch_name):
+        self.learn = learn
+        self.arch_name = arch_name
+        self.met_names = ['epoch', 'train_loss']
+        # existence of validation dataset
+        self.valid_set = self.learn.data.valid_dl.items.any()
+        if(self.valid_set):
+            self.met_names.append('valid_loss')
+
+    def on_train_begin(self, **ka):
+
+        hyp_dict = {
+            'arch_name': self.arch_name,
+            'epochs': ka['n_epochs'],
```

Comment:
Use `.get('key', default_value)` to avoid `KeyErrors`.
For multilevel, `dict.get('key1', {}).get('key2')`
https://docs.python.org/2/library/stdtypes.html#dict.get